src/main/java/gift/global/validation/validator.java

```diff
@@ -0,0 +1,34 @@
+package gift.global.validation;
+
+import gift.global.exception.BusinessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+
+/**
+ * 유효성 검증 클래스
+ */
+@Component
+public class validator {
```

Comment:
참고로 이러한 클래스, 메서드, 변수 등의 네이밍 변경은 Shift + F6을 누르면 연관된 모든 곳에 일괄변경됩니다!
혹시 모르셨다면 참고 부탁드려요ㅎㅎ
```suggestion
public class Validator {
```

Reply:
모르고 있었습니다. 감사합니다 ㅎㅎ